src/aioswitcher/api/__init__.py

```diff
@@ -183,6 +183,7 @@ async def get_state(self) -> SwitcherStateResponse:
 
         Returns:
             An instance of ``SwitcherStateResponse``.
+
```

Comment:
What's the point here? This adds line breaks in the code document.
The same goes for rows 246, 267, 360, and 710. I do not see any modifications in this file except for these five line breaks. Am I missing something? Was it because of some linting error?

Reply:
@TomerFi I just aligned those functions to the rest of the functions where there was an empty line right after the "Returns:" part

Reply:
That's good, keeping consitenency - But I don't understand why this is related to this PR.
This file wasn't event modified in this PR at all, besides these 4 line breaks in the code docs.